llvmlite/ir/instructions.py

```diff
@@ -162,7 +162,7 @@ def descr(self, buf):
 class PredictableInstr(Instruction):
 
     def set_weights(self, weights):
-        operands = [MetaDataString(self.module, "branch_weights")]
+        operands = [MetaDataString(self.module, b"branch_weights")]
```

Comment:
If so then we can simply restore the old commit but with a `.encode('ascii')` call rather than `.encode('utf-8')`. That would preserve the option 1 being the default behavior but also allow putting strings in various places.